addons/Rakugo/lib/systems/Parser.gd

```diff
@@ -67,12 +67,12 @@ var parser_regex :={
 	# jump label
 	JUMP = "^jump (?<label>{NAME})( if (?<expression>.+))?$",
 	# for setting Rakugo variables
-	SET_VARIABLE = "^(?<lvar_name>{VARIABLE}) = ((?<text>{STRING})|(?<number>{NUMERIC})|(?<rvar_name>{VARIABLE}))$",
+	SSET_VARIABLE = "^(?<lvar_name>{VARIABLE}) = ((?<text>{STRING})|(?<number>{NUMERIC})|(?<rvar_name>{VARIABLE}))$",
+	COMMENT = "^#.*$",
```

Comment:
There is a way to not use regex to read commented line and more efficient :) !

A clue : https://github.com/rakugoteam/Rakugo-Dialogue-System/blob/bcb65639c820bc414411bee3e07a4d35523e8def/addons/Rakugo/lib/systems/Parser.gd#L154

Reply:
Okey I will update code in this way.

Reply:
Just fixed with my last commit

Reply:
Just fixed with my last commit @theludovyc